File: source/dotnet/source/cmf.bus/cmf.bus.berico/IInboundEnvelopeProcessor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace cmf.bus.berico
{
    public interface IInboundEnvelopeProcessor
    {
        bool ProcessInbound(ref Envelope envelope, ref IDictionary<string, object> context);
    }
}

File: java/src/main/java/cmf/bus/berico/IInboundEnvelopeProcessor.java
package cmf.bus.berico;

import java.util.Map;

import cmf.bus.Envelope;
import cmf.bus.IDisposable;

public interface IInboundEnvelopeProcessor extends IDisposable {

    boolean processInbound(Envelope envelope, Map<String, Object> context);
}
